assets/js/components/KeyMetrics/MetricsSelectionPanel/SelectionPanelFooter.js

```diff
@@ -148,7 +149,7 @@ export default function SelectionPanelFooter( {
 	return (
 		<footer className="googlesitekit-selection-panel-footer">
 			<div className="googlesitekit-selection-panel-footer__content">
-				{ itemCountElement }
+				{ 'fullscreen' !== variation && itemCountElement }
```

Comment:
How many variations are we going to have? Just two: `default` and `fullscreen`, right? Then let's probably use `isFullscreen` instead of `variation`.